script.js

```diff
@@ -155,12 +155,12 @@ window.addEventListener("touchstart", function (e) {
 
 window.addEventListener("load", function () {
 
-    var instructions = document.createElement("section");
+    let instructions = document.createElement("section");
     instructions.className = "instructions";
-    instructions.innerHTML = "<p>Click or Touch with one finger to Start</p>"
+    instructions.innerHTML = "<p>Klicken oder tippen Sie zum Starten</p>"
```

Comment:
bare with me finding the right words.
I assume we can stick to a not too formal copy, right?
```suggestion
    instructions.innerHTML = "<p>Klicken oder mit einem Finger tippen um zu starten</p>"
```